hypershift-operator/controllers/hostedcluster/internal/platform/openstack/openstack.go

```diff
@@ -75,7 +76,12 @@ func reconcileOpenStackClusterSpec(hcluster *hyperv1.HostedCluster, openStackClu
 	// If no MachineNetwork is provided, use a default CIDR block.
 	// Note: The default is required for now because there is no CLI option to set the MachineNetwork.
 	// See https://github.com/openshift/hypershift/pull/4287
+	// When subnet is provided, machine network is required.
```

Comment:
You mean when CAPO will create the subnet, the machineNetwork is required, right?